cmd/bio-rd/main.go

```diff
@@ -17,22 +26,35 @@ import (
 	log "github.com/sirupsen/logrus"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/keepalive"
+
+	"github.com/bio-routing/bio-rd/protocols/bgp/api"
 )
 
 var (
-	configFilePath       = flag.String("config.file", "bio-rd.yml", "bio-rd config file")
+	configFilePath       *string
+	sigCh                = make(chan os.Signal, 1)
 	grpcPort             = flag.Uint("grpc_port", 5566, "GRPC API server port")
 	grpcKeepaliveMinTime = flag.Uint("grpc_keepalive_min_time", 1, "Minimum time (seconds) for a client to wait between GRPC keepalive pings")
 	metricsPort          = flag.Uint("metrics_port", 55667, "Metrics HTTP server port")
-	sigHUP               = make(chan os.Signal)
 	vrfReg               = vrf.NewVRFRegistry()
 	bgpSrv               bgpserver.BGPServer
-	runCfg               *config.Config
+	shutdownAddr         string
 )
 
+func init() {
```

Comment:
What's the point of this? If you need a certain config file name then just use the "-config.file" flag.